simple_network_sim/mixing_matrix.py

```diff
@@ -0,0 +1,132 @@
+"""A mixing matrix descibes how various groups of a population mix together.
+"""
+
+import csv
+import re
+
+AGE_RE = re.compile(r'\[(\d+),\s*(\d+)\)')
+
+class AgeRange:
+    """A helper class for an age range."""
+    def __init__(self, a, b=None):
+        """Initialiser. If b is None, it is assumed that a is a string to be
+        parsed. Otherwise, the age range is assumed to be [a, b).
+        """
+        if b is None:
+            if isinstance(a, tuple) and len(a) == 2:
+                self._lower = a[0]
+                self._upper = a[1]
```

Comment:
When are we expecting to use `AgeRange(tuple)` or `AgeRange(a=int, b=int)`?

Reply:
The tuple was given as an example in the original issue, as `returnDict[(0,5)][(5,18)]`. The two-int version I don't see a current need for at the moment, but felt trivial enough to include that it was worth writing up.